apisix/plugins/real-ip.lua

```diff
@@ -67,6 +75,22 @@ function _M.check_schema(conf)
 end
 
 
+local function addr_match(conf, ctx, addr)
+    if not conf.trusted_addresses then
```

Comment:
We already check the trusted_addresses before: https://github.com/apache/apisix/blob/025710564ac717f99fb96dd19dc1fdd1344bc7ef/apisix/plugins/real-ip.lua#L107?